infrastructure/src/main/java/org/corfudb/infrastructure/LogReplicationServer.java

```diff
@@ -26,33 +28,36 @@
  * The Log Replication Server, handles log replication entries--which
  * represent parts of a Snapshot (full) sync or a Log Entry (delta) sync
  * and also handles negotiation messages, which allows the Source Replicator
- * to get a view of the last synchronized point at the remote site.
+ * to get a view of the last synchronized point at the remote cluster.
  */
 @Slf4j
 public class LogReplicationServer extends AbstractServer {
```

Comment:
Is LogReplicationServer per site or per standby?

Reply:
a standby is a site... and yes a LRS is per standby site.